README.md

```diff
@@ -1,10 +1,11 @@
 # turniere-frontend
 
-## Prerequisites
+## Development Setup
+### Prerequisites
 
 You'll need Node.js and a package manager for Node.js (like npm or Yarn; We recommend Yarn) installed on your system in order to run this program. You can see how to install Node.js [here](https://nodejs.org/en/).
 
-## Setup the project
+### Setup the Project
```

Comment:
This is still english. Nouns are written with lower case letters.

Reply:
https://grammar.yourdictionary.com/capitalization/rules-for-capitalization-in-titles.html